src/app/hooks/useSendBtcRequest.ts

```diff
@@ -0,0 +1,62 @@
+import { Recipient, signBtcTransaction } from '@secretkeylabs/xverse-core/transactions/btc';
+import { useQuery } from '@tanstack/react-query';
+import BigNumber from 'bignumber.js';
+import { decodeToken } from 'jsontokens';
+import { useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { SendBtcTransactionOptions } from 'sats-connect';
+import useWalletSelector from './useWalletSelector';
+
+function useSendBtcRequest() {
+  const { search } = useLocation();
+  const params = new URLSearchParams(search);
+  const [recipient, setRecipient] = useState<Recipient[]>([]);
+  const requestToken = params.get('sendBtcRequest') ?? '';
+  const request = decodeToken(requestToken) as any as SendBtcTransactionOptions;
```

Comment:
is the `as any` redundant here?

Reply:
Basically The first casting (as any) is used when the type of decodeToken function's return value is not known or incompatible with the desired type. It allows treating the result as an any type, which essentially removes any type checking.The second casting (as SendBtcTransactionOptions) is attempting to explicitly assign the any type to SendBtcTransactionOptions. We know that the requestToken is compatible with SendBtcTransactionOptions since we do type checking in the sats connect package. If any attribute of SendBtcTransactionOptions is missing then the sats-connect package will throw an error. 
This was added here to be consistent with other sats-connect function screens  like `signPsbtScreen` and `getAddressScreen` since we are using the same practice in those screens as well but from my understanding , this is okay to parse responses from browser in this way

Reply:
yeah ok. feels like there should be a way to type this correctly though